iceoryx_utils/test/moduletests/test_cxx_vector.cpp

```diff
@@ -739,6 +774,19 @@ TEST_F(vector_test, EraseReturnsNullWhenElementIsInvalid)
 {
     auto i = sut.begin() + 5;
     EXPECT_THAT(sut.erase(i), Eq(nullptr));
+    i = sut0.begin() + 5;
+    EXPECT_THAT(sut0.erase(i), Eq(nullptr));
+}
+
+TEST_F(vector_test, EraseReturnsIteratorWhenElementIsValid)
```

Comment:
Here are some tests missing:
erasing with iterator:
  - erase multiple elements returns iterator
  - erase element from vector of size 1 return end()
  - erase iterator which is out of bounds

erasing with index:
  - erase multiple elements with index returns iterator
  - erase element from vector of size 1 returns end()
  - erase index which is out of bounds

Reply:
I assume by "erase multiple elements" you mean `iterator erase(const_iterator first, const_iterator last)` from std::vector? This method is not yet implemented, but could be added of course.
The other points you mentioned are done.